.cbmc-batch/source/openssl/evp_override.c

```diff
@@ -624,12 +629,17 @@ int EVP_DecryptUpdate(EVP_CIPHER_CTX *ctx, unsigned char *out, int *outl, const
     __CPROVER_assume(out_size >= 0);
     if (ctx->cipher) {
         if (ctx->padding) {
-            __CPROVER_assume(out_size <= inl + DEFAULT_BLOCK_SIZE);
+            __CPROVER_assume(out_size <= inl);
```

Comment:
Didn't we just have the same assumptions about out_size in another function? Could this be refactored into a utility?

Reply:
This assumption is specific for both of these functions. For me it's clear to have them as it is, but we can revisit this topic on a separate PR.

Reply:
OK